test/TMN.jl

```diff
@@ -265,12 +265,58 @@ set_variables(Interval{Float64}, [:x, :y], order=_order_max)
     end
     
     @testset "Tests for bounders" begin
+        beale(x, y) = (1.5 - x * (1 - y))^2 + (2.25 - x * (1 - y^2))^2 + 
+                          (2.625 - x * (1 - y^3))^2
+        rosenbrock(x, y) = 100 * (y - x^2)^2 + (1 - x)^2
+        mccormick(x, y) = sin(x + y) + (x - y)^2 - 1.5x + 2.5y + 1
+        beale_min = rosenbrock_min = 0.
+        mccormick_min = -1.9133
+            
+        @testset "Tests for linear dominated bounder" begin
```

Comment:
I  think the  tests included here involve `TaylorModelN{2,Interval{Float64},Float64}` *only*; could you add some tests for `TaylorModelN{2,Float64,Float64}`?